views/tables/ParticipantsTable/index.jsx

```diff
@@ -135,14 +136,21 @@ const ParticipantsTable = (props) => {
           />
         )
       case 'study':
+        let tooltip_title = participant[property]
+        if (participant[property] in SITE_NAMES) {
+          tooltip_title = SITE_NAMES[participant[property]]
+        }
```

Comment:
Nit
```suggestion
        const tooltip_title = participant[property]
        
        if (participant[property] in SITE_NAMES)  tooltip_title = SITE_NAMES[participant[property]]      
```

Reply:
I see that `tooltip_title`'s value is being reassigned to `const` which suggests immutability will be misleading in this context.

What is the rationale behind changing `tooltip_title` to `const`?

Reply:
That's a good question. We want to maintain readable and clean code, reassignments like these can make that a challenge.

Here is another suggestion that makes use of const, this helps in readability and it is pretty clean imo. Also, a reminder that javascript uses `[camelcase](https://developer.mozilla.org/en-US/docs/Glossary/Camel_case)`.

```suggestion
        const tooltipTitle = participant[property] in SITE_NAMES ? SITE_NAMES[participant[property]] : participant[property]
```

Now the variable is immutable, we get the value that we need, we've lessened the number of lines used and reduced mental space keeping track of all of the places that change the value of this variable. 

Let me know your thoughts..?